### ISSUE TITLE
Add information how hadolint/hadolint can be used under windows

### ISSUE THREAD

**Issue by krawinkelm** [hadolint/hadolint#513]
- [ ] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
I want to run the command from README.md on windows. I expect that this command works.

```bash
docker run --rm -i hadolint/hadolint < Dockerfile
```

### Actual behavior

Using `<` is not possible under windows /PowerShell.

### Steps to reproduce the behavior

Executed the above command in your PowerShell.

Solutions under windows:

This similiar command works under windows PowerShell. Please consider to add it to README.md

```bash
cat .\Dockerfile | docker run --rm -i hadolint/hadolint
```
